benchmark/Streamly/Benchmark/Data/Parser.hs

```diff
@@ -79,6 +79,10 @@ takeEQ value = IP.parse (PR.takeEQ value FL.drain)
 takeWhile :: MonadCatch m => Int -> SerialT m Int -> m ()
 takeWhile value = IP.parse (PR.takeWhile (<= value) FL.drain)
 
+{-# INLINE wordBy #-}
+wordBy :: MonadCatch m => Int -> SerialT m Int -> m ()
+wordBy value = IP.parse (PR.wordBy (>= value) FL.drain)
```

Comment:
Use `wordBy even` to stress the implementation for per word overhead by finding many short words.

Reply:
I'll add both, The current benchmark is the minimal, check-the-fusion benchmark.